Misc/NEWS.d/next/Library/2024-10-18-09-51-29.gh-issue-125682.vsj4cU.rst

```diff
@@ -0,0 +1 @@
+Reject unicode digits for Python implementation of :func:`json.loads`.
```

Comment:
I would clarify that this is a bug. Something more like "Conform to RFC 8259's number specification for Python implementation of `json.loads`"